source/kraken/tests/realtime_test.cpp

```diff
@@ -2966,9 +2966,7 @@ BOOST_FIXTURE_TEST_CASE(add_new_trip_and_update, AddTripDataset) {
                       pbnavitia::StopTimeUpdateStatus::ADDED);
 
     // Recall the same new trip.
-    // The old "new trip" is replace by the new, but the meta VJ is the same (don't create).
-    // The old impact is deleted too
-    // To show that it is the new GTFS-RT, schedules has changed
+    // the new trip is rejected. There is no update
```

Comment:
You changed the spirit of this test (`add_new_trip_and_update`), no?

Reply:
Nope, it is a overall test for an added trip. But if you prefer, I can extract this part, to create a test :`add_new_trip_with_id_that_already_exists`
It'll be more comfortable to read